functions/exercise11/scriptSpec.js

```diff
@@ -0,0 +1,9 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getSecondLowestAndSecondGreatestNumbers*/
+
+describe("Getting second lovest and second gratest numbers with array", function () {
+    it("[1, 2, 3, 4, 5] is 2 and 4 ", function () {
+        var value = getSecondLowestAndSecondGreatestNumbers([1, 2, 3, 4, 5]);
+        expect(value).toEqual(2, 4);
```

Comment:
You can't check it this way. You should expect object here - not two separated values.
Function can return only one thing - cant return two things at once.
So you have to pack second lowest and second largest into one thing - one object with appropriate properties. Please have a look at this discussion:
http://stackoverflow.com/questions/12272239/javascript-function-returning-an-object
And also please think how to write appropriate test to check properties of returned object.